app/assets/stylesheets/rooms.scss

```diff
@@ -1,6 +1,6 @@
 #room-card {
   border: 2px solid transparent !important;
-  width: 305px;
+  width: 311px;
```

Comment:
This should be tuned for different breakpoints.
Haven't started looking at responsiveness yet.